tests/unit/documents/test_documents_dojson_marc21.py

```diff
@@ -685,6 +685,8 @@ def test_holdings_items_to_marc21(app, marc21_record, document,
     assert result == record
 
     record = {'pid': document.pid}
+    item2_lib_sion['barcode'] = '87121336'
+    item2_lib_sion.update(item2_lib_sion, dbcommit=True, reindex=True)
```

Comment:
Did you forget to restore the initial value for item2_lib_sion at the end of the test?